client/validate.go

```diff
@@ -78,9 +78,9 @@ func (client *Client) ForgetValidationSet(accountID, name string, sequence int)
 }
 
 // ApplyValidationSet applies the given validation set identified by account and name.
```

Comment:
the doc should explain that res is return only for monitor?  but maybe it's easier/more consistent to always return it?

Reply:
Yes, I was considering this, since you agree it makes sense for enforce to be consistent I will do that, thanks.